src/rculfhash.c

```diff
@@ -568,6 +568,43 @@ unsigned int cds_lfht_fls_ulong(unsigned long x)
 #endif
 }
 
+static
```

Comment:
Follow kernel style guile.  `static` ought to be in front of the function signature, not above.  

Also, keep line a littler shorter by wrapping parameters.

Reply:
Thanks, made changes based on your suggestions.